content/static/doc/security.html

```diff
@@ -1,129 +1,89 @@
+<li>首席安全协调员： <a href="mailto:filippo@golang.org">Filippo Valsorda</a>。</li>
+<li>副安全协调员： <a href="mailto:agl@golang.org">Adam Langley</a>。</li>
+<li>若您未收到回复，请向<a href="mailto:golang-dev@googlegroups.com">golang-dev@googlegroups.com</a>发送邮件，或者使用<a href="https://groups.google.com/forum/#!forum/golang-dev">golang-dev web interface</a>。</li>
 </ul>
 
 <p>
-Please note that golang-dev is a public discussion forum.
-When escalating on this list, please do not disclose the details of the issue.
-Simply state that you're trying to reach a member of the security team.
+请注意，golang-dev是一个公开论坛，请不要透露具体信息，只需声明您正在尝试联系安全团队成员即可。
 </p>
 
-<h3>Flagging Existing Issues as Security-related</h3>
+<h3>将现有issues标记为与安全相关</h3>
 
-<p>
-If you believe that an <a href="https://golang.org/issue">existing issue</a>
-is security-related, we ask that you send an email to
-<a href="mailto:security@golang.org">security@golang.org</a>.
-The email should include the issue ID and a short description of why it should
-be handled according to this security policy.
+<p> 若您确定某个现存的 <a href="https://golang.org/issue">issue</a>有安全漏洞，请发送邮件至<a href="mailto:security@golang.org">security@golang.org</a>。邮件需要包含issue号和简短的说明，说明为什么要根据安全策略进行处理。
 </p>
 
-<h3>Disclosure Process</h3>
+<h3>安全相关问题的披露流程</h3>
 
-<p>The Go project uses the following disclosure process:</p>
+<p>Go使用以下披漏流程：</p>
 
 <ol>
-<li>Once the security report is received it is assigned a primary handler.
-This person coordinates the fix and release process.</li>
-<li>The issue is confirmed and a list of affected software is determined.</li>
-<li>Code is audited to find any potential similar problems.</li>
-<li>If it is determined, in consultation with the submitter, that a CVE-ID is
-required, the primary handler obtains one via email to
-<a href="https://oss-security.openwall.org/wiki/mailing-lists/distros">oss-distros</a>.</li>
-<li>Fixes are prepared for the two most recent major releases and the head/master
-revision. These fixes are not yet committed to the public repository.</li>
-<li>A notification is sent to the
-<a href="https://groups.google.com/group/golang-announce">golang-announce</a>
-mailing list to give users time to prepare their systems for the update.</li>
-<li>Three working days following this notification, the fixes are applied to
-the <a href="https://go.googlesource.com/go">public repository</a> and a new
-Go release is issued.</li>
-<li>On the date that the fixes are applied, announcements are sent to
-<a href="https://groups.google.com/group/golang-announce">golang-announce</a>,
-<a href="https://groups.google.com/group/golang-dev">golang-dev</a>, and
-<a href="https://groups.google.com/group/golang-nuts">golang-nuts</a>.
+<li>收到漏洞报告后，为其分配一个主理人，他负责协调漏洞修复和发布。</li>
+<li>确认该漏洞，并列出受其影响的软件。</li>
+<li>审核代码以找出任何类似的潜在漏洞。</li>
+<li>与代码提交人员协商后，若需要一个CVE-ID号，则主理人向<a href="https://oss-security.openwall.org/wiki/mailing-lists/distros">oss-distros</a>发送邮件获取。</li>
+<li>为两个主发布版和head/master修复版准备好补丁，这些补丁尚未提交到公共仓库。</li>
```

Comment:
```suggestion
<li>为最近的两个主要发行版和 head/master 版本准备了修复程序，但这些补丁尚未提交到公共仓库。</li>
```

可以讨论下， 但是英文前后的空格一定要加的